configs/benchmarks/classification/imagenet/vit-base-p16_ft-8xb128-coslr-100e-rpe_in1k.py

```diff
@@ -97,7 +97,14 @@
         weight_decay=0.05,
         model_type='vit',  # layer-wise lr decay type
         layer_decay_rate=0.65),  # layer-wise lr decay factor
-    constructor='mmselfsup.LearningRateDecayOptimWrapperConstructor')
+    constructor='mmselfsup.LearningRateDecayOptimWrapperConstructor',
+    paramwise_cfg=dict(
+        custom_keys={
+            '.ln': dict(decay_mult=0.0),
```

Comment:
We may need to update the documentation to warn the decay settings are different for different models, which shoud be carefully checked if he want to implement a new algorithm. Better to create an issue or a todo item.

Reply:
ok, I create an issue to record this